server/src/test/java/com/broadcom/lsp/cobol/TestModule.java

```diff
@@ -60,6 +61,9 @@ protected void configure() {
     bind(String.class)
         .annotatedWith(named(PATH_TO_TEST_RESOURCES))
         .toProvider(() -> ofNullable(getProperty(PATH_TO_TEST_RESOURCES)).orElse(""));
+    bind(LocaleStore.class).to(LocaleStoreImpl.class);
+    bind(MessageService.class).to(PropertiesMessageService.class);
+    bind(String.class).annotatedWith(named("resourceFileLocation")).toInstance("message/messages");
```

Comment:
"message/messages" folders ?
In our case they are not messages (JMS, Kafka, or AMQP, ... ), just raw-text, better would be: resourceBundles  or properties